plugins/ltac2/tac2entries.ml

```diff
@@ -793,15 +815,84 @@ let make_fresh_key tokens =
 
 type notation_interpretation_data =
 | Abbreviation of Id.t * Deprecation.t option * raw_tacexpr
-| Synext of bool * KerName.t * Id.Set.t * raw_tacexpr
+| Synext of bool * KerName.t * Id.Set.t * raw_tacexpr * notation_target
+
+type tac1ext = {
+  tac1ext_kn : KerName.t;
+  tac1ext_tok : sexpr list;
+  tac1ext_lev : int;
+  tac1ext_loc : bool;
+  tac1ext_depr : Deprecation.t option;
+}
```

Comment:
Maybe worth putting this in a module instead of having those `tac1ext_` everywhere